// cody-music/dist/index.js
"use strict";
var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
}) : (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    o[k2] = m[k];
}));
var __exportStar = (this && this.__exportStar) || function(m, exports) {
    for (var p in m) if (p !== "default" && !Object.prototype.hasOwnProperty.call(exports, p)) __createBinding(exports, m, p);
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.SpotifyAuthState = exports.CodyConfig = exports.PlaylistTrackInfo = exports.PlayerType = exports.SpotifyAudioFeature = exports.PlaylistItem = exports.PlayerContext = exports.PlayerDevice = exports.PaginationItem = exports.TrackStatus = exports.Track = exports.PlayerName = exports.CodyResponseType = exports.CodyResponse = exports.Artist = exports.Album = void 0;
__exportStar(require("./lib/apis"), exports);
var models_1 = require("./lib/models");
Object.defineProperty(exports, "Album", { enumerable: true, get: function () { return models_1.Album; } });
Object.defineProperty(exports, "Artist", { enumerable: true, get: function () { return models_1.Artist; } });
Object.defineProperty(exports, "CodyResponse", { enumerable: true, get: function () { return models_1.CodyResponse; } });
Object.defineProperty(exports, "CodyResponseType", { enumerable: true, get: function () { return models_1.CodyResponseType; } });
Object.defineProperty(exports, "PlayerName", { enumerable: true, get: function () { return models_1.PlayerName; } });
Object.defineProperty(exports, "Track", { enumerable: true, get: function () { return models_1.Track; } });
Object.defineProperty(exports, "TrackStatus", { enumerable: true, get: function () { return models_1.TrackStatus; } });
Object.defineProperty(exports, "PaginationItem", { enumerable: true, get: function () { return models_1.PaginationItem; } });
Object.defineProperty(exports, "PlayerDevice", { enumerable: true, get: function () { return models_1.PlayerDevice; } });
Object.defineProperty(exports, "PlayerContext", { enumerable: true, get: function () { return models_1.PlayerContext; } });
Object.defineProperty(exports, "PlaylistItem", { enumerable: true, get: function () { return models_1.PlaylistItem; } });
Object.defineProperty(exports, "SpotifyAudioFeature", { enumerable: true, get: function () { return models_1.SpotifyAudioFeature; } });
Object.defineProperty(exports, "PlayerType", { enumerable: true, get: function () { return models_1.PlayerType; } });
Object.defineProperty(exports, "PlaylistTrackInfo", { enumerable: true, get: function () { return models_1.PlaylistTrackInfo; } });
Object.defineProperty(exports, "CodyConfig", { enumerable: true, get: function () { return models_1.CodyConfig; } });
Object.defineProperty(exports, "SpotifyAuthState", { enumerable: true, get: function () { return models_1.SpotifyAuthState; } });
